README.md

```diff
@@ -211,5 +313,4 @@ In general, in order to add new commands, you need to add these files:
 	- This file will contain the methods needed to transform the extracted data into a form that is suitable for BigQuery.
 	- The struct definition for the transformed object should be stored in `schemas.go` in the `internal/transform` folder.
 
-A good number of common methods are already written and stored in the `util` package.
-
+A good number of common methods are already written and stored in the `util` package.
```

Comment:
is it worth listing any of common methods?

Reply:
I don't think so. There are a lot of functions and I think most of them are pretty self explanatory or have a comment/description